mlir/test/lit.site.cfg.py.in

```diff
@@ -54,6 +54,11 @@ config.random_data = "@MLIR_RANDOM_DATA@"
 config.rocmlir_gen_flags = "@ROCMLIR_GEN_FLAGS@"
 config.populate_validation = "@MLIR_POPULATE_VALIDATION@"
 config.rocm_path = "@ROCM_PATH@"
+config.rocmlir_common_python_tests_utils = "@ROCMLIR_COMMON_PYTHON_TESTS_UTILS@"
+
+# Add common python test utils
+sys.path.append(config.rocmlir_common_python_tests_utils)
```

Comment:
Is there a way to `with` this to make it a temporary adjustment to the import path?

Reply:
Hi @krzysz00, I did quite get what you mean.

`sys.path.append(config.rocmlir_common_python_tests_utils)` - is a temporary adjustment to the system path.

Do you mean whether we can do something to just import `common_utils` without modifying the system path? - i.e.,

```python
from arch_features import get_arch_features
```

I tried to come with something which doesn't require to modify the system path but nothing worked for me. Eventually, I came up with `sys.path.append`

Reply:
I think he's looking for a context manager to append to sys.path and then undo it.  Though since it's only for the duration of the test-discovery phase, it may not matter much.

Reply:
> I think he's looking for a context manager to append to sys.path and then undo it. Though since it's only for the duration of the test-discovery phase, it may not matter much.

I can see. Thanks for explaining me. At this moment, I don't have any idea how to implement it.